site/docs/components/cookie_compliance.mdx

```diff
@@ -0,0 +1,136 @@
+</LargeParagraph>
+
+
+## Principles
+
+### Cookie clauses in English (EN)
+
+#### Banner text (accordion closed)
+**Headline:** [Service name] uses cookies
+
+**Bodycopy:** This website uses essential cookies to ensure site performance and for basic analytics. In addition, we use uses targeting cookies to improve the user experience, for analytics and showing targeted content. By continuing to use the site, you agree to the use of essential cookies.
+
+#### Banner text (accordion open)
+**Headline:** [Service name] uses cookies
+
+**Bodycopy:** This website uses essential cookies to ensure site performance and for basic analytics. In addition, we use uses targeting cookies to improve the user experience, for analytics and showing targeted content. By continuing to use the site, you agree to the use of essential cookies.
+
+**Subhead:** Information about the cookies used on the website
+
+**Bodycopy:** The cookies used on this website are classified according to their purpose. You can find information about each category below and allow or deny the use of cookies.
+
+**H:** Mandatory cookies
+
+**B:** The use of mandatory cookies cannot be denied. They enable the operation of the site and affect E.g. the usability of the site.
+(Checkbox + duration + information about the cookie)
+
+**H:** Optional cookies
+
+**B:** You can accept or decline optional cookies.
+(Checkbox + duration + information about the cookie)
+
+### UI components
+
+#### 1. Language selection
+The default UI componentes and cookie descriptions are delivered in Finnish, Swedish and English.
+
+#### 2. Cookie selection
+The cookies are divided to mandatory and optional cookies. The mandatory cookies are displayed "grayed-out" to visually support the fact that they cannot be unselected.
+
+#### Default checkboxes for cookie consent
+
+**Mandatory cookies** (visually grayed-out / disabled)
+
+- Login cookies: A mandatory cookie that allows the visitor to visit the site.
+- Language cookies: We store the interface language you have selected.
+
+**Optional cookies**
+
+- Statistic cookies: The information collected by statistical cookies is used to develop the website further.
+- Preference cookies: Preference cookies customize E.g. the look and feel of a site based on a user's choices.
+- Marketing cookies: Marketing cookies can be applied to target specific content to users.
+
+#### 3. Consent confirmation buttons
+- Accept all cookies
+- Only mandatory
+- Continue with selected (TBD)
+- Opt out (TBD)
+- Close (cross icon) -button
+
+## Usage
+
+### When to use
+ - When you need to inform users about cookie usage. 
+
+### When not to use
+ - For any other usage except informing about cookie usage.
+
+### When to use each banner type?
+| Site uses cookies              | Cookie type                    | What to do and how to inform the user?                                                                              | Example
```

Comment:
This example column is slightly confusing. It states which elements should be included in the consent banner - so should it be named something else? "Elements to include" or similar?

Reply:
Changed to suggested